CHANGELOG.md

```diff
@@ -6,6 +6,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0] Hyperion - 2021-11-08 [:warning:]
+Reorg-resistance for the Ethereum backend and support for external Watchtowers.
+
+### Added
+
+- :sparkles: **Ethereum backend: Reorg resistance** [#19](https://github.com/hyperledger-labs/go-perun/issues/19): The Ethereum backend now lets the user specify after how many blocks a transaction or an event should be considered confirmed.
+- :sparkles: **Watcher interface** [#172](https://github.com/hyperledger-labs/go-perun/issues/172): The watcher logic is now injectable. The adjudicator takes a watcher instance as a setup parameter. This enables using remote watcher services. A local watcher implementation is provided.
+
+### Changed
+
+- :arrow_up: Update go to v1.17 and go-ethereum to v1.10.8.
```

Comment:
We usually take a release as an opportunity to update geth to the latest version (and maybe the other dependencies, too). Maybe update to [geth 1.10.12](https://github.com/ethereum/go-ethereum/releases/tag/v1.10.12) (and see if the tests still run).

Reply:
Done